product_special_type/__manifest__.py

```diff
@@ -1,25 +1,21 @@
-# -*- coding: utf-8 -*-
 # Copyright 2012-2018 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
-    'name': 'Product Special Types',
-    'version': '10.0.1.0.0',
-    'author': 'Camptocamp,Odoo Community Association (OCA)',
-    'license': 'AGPL-3',
-    'category': 'Product',
-    'development_status': 'Beta',
-    'maintainers': ['rousseldenis'],
-    'summary':
-    '''
+    "name": "Product Special Types",
+    "version": "16.0.1.0.0",
```

Comment:
@acsonefho Could you follow migration guide ? https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#technical-method-to-migrate-a-module-from-150-to-160-branch

Version change should appear in a migration commit and not in pre-commit stuff. Thanks